src/plugins/intel_npu/tests/functional/behavior/npuw/behavior_tests.cpp

```diff
@@ -53,209 +61,470 @@ TEST_F(BehaviorTestsNPUW, TestInfrastructureIsCorrect) {
 }
 
 TEST_F(BehaviorTestsNPUW, CompilationIsSuccessful) {
+    model = model_generator.get_model_with_one_op();
```

Comment:
Shouldn't this model be a default one since you write this line many times?

Reply:
I don't think so. It is here to differentiate tests for model with one operation and tests for model with repeated blocks